shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -564,15 +580,16 @@ export default {
           <template #group-by="{group}">
             <div class="pool-row" :class="{'has-description':group.ref && group.ref.nodeTemplate}">
               <div v-trim-whitespace class="group-tab">
-                <div v-if="group.ref" v-html="t('resourceTable.groupLabel.nodePool', { name: group.ref.spec.hostnamePrefix, count: group.ref.scale}, true)">
+                <div v-if="group.ref" v-html="t('resourceTable.groupLabel.nodePool', { name: group.ref.spec.hostnamePrefix}, true)">
                 </div>
                 <div v-else v-html="t('resourceTable.groupLabel.notInANodePool')">
                 </div>
                 <div v-if="group.ref && group.ref.nodeTemplate" class="description text-muted text-small">
                   {{ group.ref.providerDisplay }} &ndash;  {{ group.ref.providerLocation }} / {{ group.ref.providerSize }} ({{ group.ref.providerName }})
                 </div>
               </div>
-              <div v-if="group.ref" class="right">
+              <div v-if="group.ref" class="right group-header-buttons">
+                <MachineSummaryGraph :row="poolSummaryInfo[group.ref]" :horizontal="true" class="mr-20" />
```

Comment:
This pages shows rke1 nodes or rke2 machines, so there's two lists here. In order to show the same graph for rke2 clusters the component just needs adding to the tab above (`<Tab v-if="showMachines"`)

Reply:
i tried adding it in, but there's a minor styling issue
![image](https://user-images.githubusercontent.com/18697775/173107735-8108d45e-99b9-4f89-9ece-2303683f888e.png)

Reply:
What was the styling issue? I've updated, as per the suggestion - looks good to me.

Reply:
I was seeing the plus button flush to the right side of the table